projects/storefrontstyles/scss/layout/page-templates/_product-detail.scss

```diff
@@ -15,7 +15,12 @@
 
         cx-product-images {
           grid-column: 1;
-          grid-row: 1 / span 6;
+          @include forVersion(3) {
```

Comment:
I don't think you need to wrap this one because this is what is currently in develop. Only the change for 3.1 should be wrapped.